stable/elasticsearch-exporter/templates/deployment.yaml

```diff
@@ -37,6 +37,11 @@ spec:
         runAsUser: 1000
       containers:
         - name: {{ .Chart.Name }}
+          env:
```

Comment:
In case of `env: {}`, this will produce an invalid yaml. Better add condition and possibly comment out env in values.yaml

Reply:
Hey @desaintmartin, thanks for the review - I've added in a condition for cases where no `env` is specified - let me know if it looks ok to you.